queries.sql

```diff
@@ -80,4 +80,24 @@ ROLLBACK;
   -- List all animals owned by Dean Winchester that haven't tried to escape.
   SELECT animals.name, owners.full_name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE escape_attempts = 0 AND owners.id = 5;
   -- Who owns the most animals?
-  SELECT owners.full_name, COUNT(*) FROM owners LEFT JOIN animals ON owners.id = animals.owner_id GROUP BY owners.full_name ORDER BY count DESC;
\ No newline at end of file
+  SELECT owners.full_name, COUNT(*) FROM owners LEFT JOIN animals ON owners.id = animals.owner_id GROUP BY owners.full_name ORDER BY count DESC;
```

Comment:
- [ ] Please add a limit to return row of data showing the last animal seen by William. You can add `LIMIT 1` after `DESC` to return the last animal seen by William because currently, you are returning more animals as seen below.
![image](https://github.com/fatmahussein/vet_clinic/assets/45460192/043ba1d7-0e35-461b-ae52-a2e618f18540)